qemu/tests/thin_write_in_qemu_img_commit.py

```diff
@@ -0,0 +1,61 @@
+import logging
+import json
+
+from virttest import data_dir
+from virttest import utils_numeric
+from virttest.qemu_io import QemuIOSystem
+from virttest.qemu_storage import QemuImg
+
+from qemu.tests.qemu_disk_img import QemuImgTest
+from qemu.tests.qemu_disk_img import generate_base_snapshot_pair
+
+
+def run(test, params, env):
+    """
+    Check thin write in qemu-img commit.
+
+    :param test: Qemu test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    def _get_image_object(tag):
+        """Get QemuImg object by tag."""
+        img_param = params.object_params(tag)
+        img = QemuImg(img_param, img_root_dir, tag)
+        return img
```

Comment:
could just return.

Reply:
The `QemuImg` has to be returned, otherwise it can not be referenced later.

Reply:
I mean variable `img` is redundant.